### PR TITLE
Update EmbeddedApp.md

### PR DESCRIPTION
**Pull request by lvyile**
L34~L39  <==> L126   should be same

### CODE REVIEW

**Comment by vjeux** [comment]
Thanks, can you sign the CLA, rebase and i'll pull this in :)

**Comment by lvyile** [comment]
cla signed up.

File: docs/EmbeddedApp.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ The installation process also requires [Node.js](http://nodejs.org).
 
 ## Create Your React Native App
 
-First, enter React Native's pod root directory and create **index.ios.js** inside a directory `ReactComponent`.
+First, create a directory `ReactComponent`, enter the directory and create **index.ios.js** inside .
```
Comment (sahrens) [inline docs/EmbeddedApp.md]: can you change index.ios.js to use backticks as well?

Comment (lvyile) [inline docs/EmbeddedApp.md]: sure, done

**Comment by vjeux** [comment]
It has merge conflicts :( Can you rebase and i'll pull it in. Thanks!

**Comment by lvyile** [comment]
I'm learning 'git rebase' via SourceTree.app ( #549 is my first rebase in my life, lol )
Now I'm done.
Hope everything goes well. 
Thx guys.
screen shot 2015-04-02 at 10 07 28 pm

**Comment by vjeux** [comment]
Didn't seem to work :( Maybe you want to abandon this one and recreate a new one with just your change?

**Comment by lvyile** [comment]
ok, got it. Let me try : )

**Pull request closed without merging**